### PR TITLE
Add 48Club BSC RPCs

### PR DESCRIPTION
**Pull request by 48ClubIan**
Remove old bnb48 domain
add new RPC addresses

Link the service provider's website (the company/protocol/individual providing the RPC):
https://www.48.club/

Provide a link to your privacy policy:
https://docs.48.club/buidl/infrastructure/terms-of-use

### CODE REVIEW

**Comment by 48ClubIan** [comment]
@realdealshaman Hello, just updated the new PR.

**Comment by 0xngmi** [comment]
Can you elaborate on what you mean by?
>  Also notice that we don't actively purge user feetprint in lower-level protocol.

Also, should change feetprint to footprint

**Comment by 48ClubIan** [comment]
> Can you elaborate on what you mean by the following
> 
> > Also notice that we don't actively purge user feetprint in lower-level protocol.
Like we don't modify original program to stop logging or purge generated logs of nginx/httpd/geth_bsc etc.
> 
> Also, should change feetprint to footprint
Thank you for reminding. It's fixed now.

File: constants/extraRpcs.js [hunk 1]
```diff
@@ -373,8 +385,7 @@ export const extraRpcs = {
         tracking: "yes",
         trackingDetails: privacyStatement.nodereal,
       },
-      "https://rpc-bsc.bnb48.club",
-      "https://bsc-mainnet.rpcfast.com?api_key=xbhWBI1Wkguk8SNMu1bvvLurPGLXmgwYeC4S6g2H7WdwFigZSmPWVZRxrskEQwIf",
+      "https://bsc-mainnet.rpcfast.com",
```
Comment (0xngmi) [inline constants/extraRpcs.js:388]: this change is wrong

**Pull request merged**